lib/render/set-claim.php

```diff
@@ -1,22 +1,32 @@
 <?php
 
 use RA\ClaimFilters;
+use RA\ClaimSetType;
 use RA\ClaimSorting;
 
 /**
- * Creates the New Set Claims component.
+ * Creates the Set Claims component.
  */
-function renderNewClaimsComponent(int $count): void
+function renderClaimsComponent(int $count, int $claimFilter = ClaimFilters::Default, int $claimSort = ClaimSorting::ClaimDateDescending): void
```

Comment:
I'm really struggling with this change. About half of the function is shared, and the other half is conditionally A or B based on a parameter that can have values other than A or B.

I really think this should probably be two separate functions despite the fact that's there's a decent amount of redundancy between them.

Reply:
I was a bit on the fence about it as well but figured less code was better. I'll split it out though as it's easier to read that way